_posts/2024-08-21-pyopensci-monumental-growth-2024.md

```diff
@@ -0,0 +1,418 @@
+    </figcaption>
+</figure>
+
+### pyOpenSci and iterative, data-driven program design
+
+pyOpenSci's development and design is data-driven. I collected survey data
+from scientists and Pythonistas at meetings and conferences to identify community pain points and needs. I used this data to inform the iterative development of pyOpenSci programs, events, and resources and to define pyOpenSci's mission, vision, values, and structure.
+
+The survey data I've collected shows what I've seen in the classroom-- scientists face many challenges when
+processing, analyzing, and sharing their workflows. While reproducible
+science is critical to accelerating science, sharing and publishing code is hard.
+
+This blog post includes quotes and data I've collected over the past 5+ years; these data have shaped the vibrant community of practice that pyOpenSci is today.
+
+<figure>
+    <a href="/images/pyopensci-general/pyopensci-iterative-evaluation-program-design.png">
+    <img src="/images/pyopensci-general/pyopensci-iterative-evaluation-program-design.png" style="max-width:60%" class="align-center" alt="A round graphic that shows the cycle of iterative program development. The pyOpenSci logo is at the top left. The graphic title reads iterative program development. The first step says, 'A good idea gets better and better', the second says  'survey community needs,' the third says, 'evaluation based program design,' and the 4th says, 'evaluate results and make iterative improvements'. In the center is th">
+    </a>
+    <figcaption> Iterative data-driven program design is a fusion of evaluation and program development that utilizes community feedback to build a program using an iterative and community-responsive approach. You test approaches and collect data about the program's effectiveness. This type of development allows an organization to grow dynamically, following community needs.
+    </figcaption>
+</figure>
+
+<div class="notice" markdown ="1">
+
+### Responses to the question: *How could pyOpenSci help you with your science, code, and software?*
+{: .not_toc}
+
+{% include pyos-blockquote.html quote="[*I want to know...*] What are the best practices for sharing the code?" author="Anonymous" event="AGU 2019 Townhall" class="highlight" event="2019 AGU Town Hall" %}
+
+{% include pyos-blockquote.html quote="[*I want to... *] Streamline the development of good quality, socially responsible, and easily shareable software." author="Anonymous" event="AGU 2019 Townhall" class="highlight magenta" %}
+
+{% include pyos-blockquote.html quote="[*I want more *] Bullet-proof, well-documented packages for Earth science." author="Anonymous" event="AGU 2019 Townhall" class="highlight" %}
+
+Many earth scientists attend AGU. It's a different crowd than who you meet
```

Comment:
I have no opinion on this! If you think it's better to capitalize, we should do that. Earth scientists are referenced as a domain I'm discussing here. For instance, a student studying earth science. It's not a job title specifically. If domain areas are supposed to be capitalized, we should do so. 

It might be useful to write down some of this. 

Then we can see how much of it someone could add to vale? What do you think about that? I won't remember it all. But I would follow a guide or a linter's suggestions.